app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/TicketScanValidationActivity.java

```diff
@@ -0,0 +1,31 @@
+package de.tum.in.tumcampusapp.component.ui.ticket;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.os.Bundle;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.other.generic.activity.BaseActivity;
+
+public class TicketScanValidationActivity extends BaseActivity {
+
```

Comment:
Is this class used at all? If not, please delete the class and the entry in the manifest.

Reply:
@saileru is right. Sorry, my bad :man_shrugging: